client/components/happychat/style.scss

```diff
@@ -52,6 +52,10 @@
 	right: 12px;
 	font-size: 14px;
 	display: none;		/* disabled */
+	box-sizing: border-box;
+
+	transform: translateZ(0);
```

Comment:
went poking and it appears that autoprefixer doesn't do anything fancy with it. I'm happy with the backface / translate tricks for now.